db/migrate/20180721114724_change_comments_to_utf8mb4.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeCommentsToUtf8mb4 < ActiveRecord::Migration[5.2]
+   def change
+    # changing table that will store unicode execute:
+    execute "ALTER TABLE comments CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
+    # changin string/text column with unicode content execute:
+    execute "ALTER TABLE comments MODIFY comment VARCHAR(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin"
```

Comment:
Do we really feel comfortable just truncating the field?
Let's not truncate the field just yet (line 6), it shouldn't be necesssary.

Also let's use collation `utf8mb4_0900_ai_ci` it seems to be preferred (we run MariaDB 10.2 which supports this).